docs/guides/CellGeolocations.rst

```diff
@@ -44,13 +44,13 @@ This should also be the preferred aproach when developing an IoT product that ha
 If a device knows its cell tower it is mormally safe to assume that it has an internet connection in order to request the approximate location from the cloud.
 This allows to offload expensive calculations to the cloud and reduce the resource usage on the device.
 
-In the Asset Tracker Cloud Example every time a device reports a GPS fix, that position is stored together with the cell id.
+In the nRF Asset Tracker every time a device reports a GPS fix, that position is stored together with the cell id.
 Over time this builds a cell location database that is independet from a third party API and up to date.
-It is used as the primary mean of geolocation cells in the Asset Tracker Cloud Example.
+It is used as the primary mean of geolocation cells in the nRF Asset Tracker.
```

Comment:
```suggestion
It is used as the primary means of geolocated cells in the nRF Asset Tracker.
```

Reply:
```suggestion
It is used as the primary means of geolocating cells in the nRF Asset Tracker.
```
It is about the process of acquiring the geolocation of cells, so `geolocating` sounds better to me.